runtimes/pythonrt/py-sdk/autokitteh/gemini.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import google.generativeai as genai
+
+from .connections import check_connection_name
+from .errors import ConnectionInitError
+
+
+def gemini_client(connection: str, **kwargs) -> genai.GenerativeModel:
+    """Initialize a genai client, based on an AutoKitteh connection.
+
+    API reference:
+    https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai/GenerativeModel.md
+
+    Args:
+        connection: AutoKitteh connection name.
+
+    Returns:
+        An initialized GenerativeModel instance.
+
+    Raises:
+        ValueError: AutoKitteh connection name is invalid.
+        ConnectionInitError: AutoKitteh connection was not initialized yet.
+
+    """
+    check_connection_name(connection)
+
+    # Configure API key
```

Comment:
https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar

```suggestion
    # Set the API key, if possible.
```